WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/AdaptiveImage.swift

```diff
@@ -0,0 +1,17 @@
+import SwiftUI
+
+struct AdaptiveImage: View {
+    @Environment(\.colorScheme) var colorScheme
```

Comment:
Do we need to add scale here in the same way we do for the `AdaptiveAsyncImage`?

Reply:
Not here, because in this case we already have the `UIImage`, so the scale is already fixed. We could use the scale at the point it's created from the data (i.e. `UIImage(data: base64Data, scale: 3.0)` when we're making the image... but since it's not required for this API, assuming a 1x scale and resizing to fit its frame may be safer. WDYT?